docs/src/components/productX/XHero.tsx

```diff
@@ -292,12 +239,6 @@ export default function XHero() {
                   rowGroupingCounter += 1;
                   return rowGroupingCounter === 3;
                 }}
-                rowGroupingModel={rowGroupingModel}
-                onRowGroupingModelChange={(model) => setRowGroupingModel(model)}
-                aggregationModel={aggregationModel}
-                onAggregationModelChange={(newModel) => setAggregationModel(newModel)}
-                columnVisibilityModel={columnVisibilityModel}
-                onColumnVisibilityModelChange={(newModel) => setColumnVisibilityModel(newModel)}
```

Comment:
There's no need to use controlled models for this, `initialState` should be enough here